src/topic.ts

```diff
@@ -500,7 +558,7 @@ export class Topic {
    *   }
    * });
    */
-  publisher(options?) {
+  publisher(options?: PublisherCallOptions) {
```

Comment:
I know ts will probably figure it out, but maybe we should be explicit that this method returns a `Publisher` instance